cloud_storage/public/js/cloud_storage.bundle.js

```diff
@@ -1 +1,3 @@
 import './overrides'
+
+frappe.Docx = require('docx-preview')
```

Comment:
This seems to be required, otherwise the browser throws a "Cannot use import statement outside a module" error in the console.